epochStart/metachain/validators.go

```diff
@@ -67,15 +67,15 @@ func NewValidatorInfoCreator(args ArgsNewValidatorInfoCreator) (*validatorInfoCr
 }
 
 // CreateValidatorInfoMiniBlocks creates the validatorInfo miniblocks according to the provided validatorInfo map
-func (vic *validatorInfoCreator) CreateValidatorInfoMiniBlocks(validatorsInfo map[uint32][]*state.ValidatorInfo) (block.MiniBlockSlice, error) {
+func (vic *validatorInfoCreator) CreateValidatorInfoMiniBlocks(validatorsInfo state.ShardValidatorsInfoMapHandler) (block.MiniBlockSlice, error) {
 	if validatorsInfo == nil {
 		return nil, epochStart.ErrNilValidatorInfo
 	}
 
 	miniblocks := make([]*block.MiniBlock, 0)
 
 	for shardId := uint32(0); shardId < vic.shardCoordinator.NumberOfShards(); shardId++ {
-		validators := validatorsInfo[shardId]
+		validators := validatorsInfo.GetShardValidatorsInfoMap()[shardId]
```

Comment:
maybe split into 2 lines? `validatorsMap =  validatorsInfo.GetShardValidatorsInfoMap()` and then `validators := validatorsMap[shardId]`

Also, you can use the same variable on L91